github-webhooks-to-s3/variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "owner" {
+  type        = "string"
+  description = "Owner for tagging and naming."
+}
+
+variable "project" {
+  type        = "string"
+  description = "Project for tagging and naming."
+}
+
+variable "service" {
+  type        = "string"
+  description = "Service for tagging and naming."
+}
+
+variable "env" {
+  type        = "string"
+  description = "Env for tagging and naming."
+}
+
+variable "fqdn" {
+  type        = "string"
+  description = "The fqdn to expose the api gateway as"
+}
+
+variable "certificate_arn" {
+  type        = "string"
+  description = "A certificate in us-east-1 for var.fqdn"
+}
+
+variable "route53_zone_id" {
+  type        = "string"
+  description = "The route53 zone id for fqdn's domain"
+}
+
+variable "lambda_source_s3_bucket" {
+  type        = "string"
+  description = "The s3 bucket where to find the lambda executable"
+  default     = "shared-infra-prod-assets"
```

Comment:
TODO @edulop91 check this bucket

Reply:
Were you able to check this bucket?

Reply:
yeah! i'm working on moving some of this data to a new public bucket